front/cypress/e2e/project_description_builder/language_switch.cy.ts

```diff
@@ -32,13 +38,13 @@ describe('Project description builder language switch', () => {
   });
 
   beforeEach(() => {
+    switchLocale('en');
     cy.setAdminLoginCookie();
   });
 
   after(() => {
     cy.visit(`/projects/${projectSlug}`);
-    cy.get('.e2e-language-dropdown-toggle').click({ force: true });
```

Comment:
The content builder didn't remove the citizen navigation bar, it was just hiding it. Maybe in this epic, we could do the same. So, all e2e tests would continue to pass. But I assume it's a bit dirty.

Reply:
> Maybe in this epic, we could do the same. So, all e2e tests would continue to pass. But I assume it's a bit dirty.

Not sure what you mean exactly. Which tests would continue to pass?

Reply:
The content builder takes up an entire screen. Previously, it hovered the citizen navigation bar (this line worked because of `force: true`). Now the citizen navigation bar is gone in the back office, that's why this line stopped working.

I thought that maybe instead of hiding the citizen navbar in the back office, we could put it in the background. I supposed it could make fixing tests easier. But they're fixed now in a proper way, so probably not worth considering anymore.